files/models.py

```
@@ -1981,6 +1982,40 @@ def media_save(sender, instance, created, **kwargs):
     instance.transcribe_function()
 
 
+def _schedule_storage_usage_refresh_for_media(media_id):
+    try:
+        from .storage_usage import schedule_refresh_media_storage_usage
+
+        schedule_refresh_media_storage_usage(media_id)
+    except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)